tox.ini

```diff
@@ -1,14 +1,15 @@
 [tox]
-envlist = py3{5,6,7,8,9}
+envlist = py3{6,7,8,9,10}
 
 [testenv]
 deps =
   -r requirements/develop.txt
   pip>=20.0
+passenv = RUNNER_OS
```

Comment:
Might be worth being a bit more explicit rather than naming files in git based on GitHub's OS naming scheme (including case)?  Also, this requires setting this env var locally in accordance to GitHub's naming to be able to successfully run tox locally so perhaps a cli option or other mechanism would be better?

Reply:
Tox is just so long to run that I use it only for the CI. But indedd, on a local computer, this beomes an issue. I am quite unfamiliar with tox, what would be a good aproach for this ?

Reply:
I guess "the tox way" would actually be an environment factor.  Something like below.  Then running tox you would `tox -v -e py39-windows`, or in the CI config something like `tox -v -e ${{ matrix.python.tox }}-${{ matrix.os.tox }}`.

```python
[testenv]
deps =
  -r requirements/develop.txt
  pip>=20.0
setenv:
  windows: OS_FILE_MARKER = windows
  linux: OS_FILE_MARKER = linux
commands = 
  pip install PyQt5==5.15.* PyQt3D==5.15.* PyQtChart==5.15.*  PyQtDataVisualization==5.15.* PyQtNetworkAuth==5.15.* PyQtPurchasing==5.15.* PyQtWebEngine==5.15.*
  mypy --show-error-codes -p PyQt5-stubs
  stubtest --allowlist {toxinidir}/stubtest.allowlist --allowlist {toxinidir}/stubtest.allowlist.to_review --allowlist {toxinidir}/stubtest.allowlist.{env:OS_FILE_MARKER} PyQt5
  pytest --capture=no --verbose {posargs}
```

Reply:
done as described.